tests/Invariants.hs

```diff
@@ -0,0 +1,16 @@
+module Invariants (invariantTests) where
+
+
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit
+import Language.Haskell.Exts
+
+
+invariantTests :: TestTree
+invariantTests = testGroup "Unit tests"
+    [ testCase "Pretty printing with layout = PPInLine" $ do
+        let decl = "h x = f (filter ('a' ==) x)"
```

Comment:
Ok. 
Just let me know if the changes to .gitignore and .cabal files are ok, or what to do with them and I will prepare new PR.